apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.sql

```diff
@@ -75,3 +75,11 @@ FROM
   ) AS ssq ON ssq.sharing_set_id = ss.id
 WHERE
   ss.course_id = $course_id;
+
+-- BLOCK select_topics_by_course_id
+SELECT
+  t.*
+FROM
+  topics AS t
+WHERE
+  t.course_id = $course_id;
```

Comment:
Nit, you can get away without aliases for super simple queries like this.

```suggestion
SELECT
  *
FROM
  topics
WHERE
  course_id = $course_id;
```

Reply:
Good to know! Did not use aliases when moving the query to `models/topics` as described in the other comment.